secio/src/error.rs

```diff
@@ -79,7 +79,7 @@ impl Into<io::Error> for SecioError {
     fn into(self) -> io::Error {
         match self {
             SecioError::IoError(e) => e,
-            e => io::Error::new(io::ErrorKind::BrokenPipe, error::Error::description(&e)),
+            e => io::Error::new(io::ErrorKind::BrokenPipe, e.to_string()),
```

Comment:
unrelated change for this bug fixing? suggest to sperate it to another PR.

Reply:
rustc 1.42 check warning

Reply:
> rustc 1.42 check warning

this repo's rust-toolchain file is still 1.41, I think that we can upgrade it with this fix in other PR.

Reply:
ok, reset it now